.github/workflows/main.yml

```diff
@@ -75,11 +75,11 @@ jobs:
 
     - name: Install ch_util
       run: |
-        pip install --use-deprecated=legacy-resolver ./ch_util
```

Comment:
Now you've fixed everything, can this be removed, along with the explicit clone of the ch_util two blocks above?